//
//  amtp_event.hpp
//  Amtp
//
//  Created by jiangwei on 2020/3/2.
//  Copyright Â© 2020 jiangwei. All rights reserved.
//

#ifndef amtp_event_hpp
#define amtp_event_hpp

#include <stdio.h>
#include <memory>
#include "../../../jwumq/jwumq_message.hpp"
#include "../../libamtpa_define.h"
#include "../../../license/libamtpa_license.hpp"


using namespace std;


class amtp_event
{
public:
	amtp_event();
	~amtp_event(void);
	
	unique_ptr<JwumqMessage> data(REPORT_EVENT_STRU * s, unsigned char *token, LIC_CONF_T load_conf, uint32_t sn, string src_id, string des_id);
	
private:
	
public:
	
private:
	
};

#endif /* amtp_event_hpp */
